### PR TITLE
ДЗ 1

### PR DESCRIPTION
**Pull request by ppzhuk**
Проверьте, пожалуйста, задание.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -21,4 +21,5 @@ android {
 
 dependencies {
     compile 'com.android.support:appcompat-v7:23.1.1'
-}
+    compile 'com.squareup.picasso:picasso:2.5.2'
+}
```
Comment (thevery) [inline app/build.gradle:25]: newline should not be removed

File: app/src/main/java/com/csc/lesson1/MainActivity.java [hunk 2]
```diff
@@ -2,11 +2,25 @@
...
 
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        String url =
+                "https://compscicenter.ru/media/cache/04/af/04af00c38359378a149700f9f9786f27.jpg";
```
Comment (thevery) [inline app/src/main/java/com/csc/lesson1/MainActivity.java:17]: ha-ha!

File: app/src/main/java/com/csc/lesson1/MainActivity.java [hunk 3]
```diff
@@ -2,11 +2,25 @@
...
+import com.squareup.picasso.Picasso;
 
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        String url =
```
Comment (thevery) [inline app/src/main/java/com/csc/lesson1/MainActivity.java:16]: String should be const

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+            android:textAppearance="?android:attr/textAppearanceMedium"
+            android:text="@string/full_name"
+            android:layout_gravity="center"
+            android:id="@+id/full_name"
+            android:padding="5dp" />
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textAppearance="?android:attr/textAppearanceMedium"
```
Comment (thevery) [inline app/src/main/res/layout/activity_main.xml:41]: Nice!

File: app/src/main/java/com/csc/lesson1/MainActivity.java [hunk 5]
```diff
@@ -2,11 +2,25 @@
...
+                "https://compscicenter.ru/media/cache/04/af/04af00c38359378a149700f9f9786f27.jpg";
+        ImageView photo = (ImageView)findViewById(R.id.photo);
```
Comment (thevery) [inline app/src/main/java/com/csc/lesson1/MainActivity.java:18]: Reformat code command is your friend!

File: app/src/main/res/layout-land/activity_main.xml [hunk 6]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.csc.lesson1.MainActivity"
+    android:orientation="horizontal">
+
+    <FrameLayout
```
Comment (thevery) [inline app/src/main/res/layout-land/activity_main.xml:9]: Photo is still small so this layout is pretty useless

File: app/src/main/res/layout/activity_main.xml [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+        android:layout_height="0dp"
+        android:layout_weight="1">
+
+        <ImageView
+            android:layout_gravity="center"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_margin="25dp"
+            android:id="@+id/photo"
+            android:contentDescription="@string/full_name"/>
```
Comment (thevery) [inline app/src/main/res/layout/activity_main.xml:20]: Nice but useless

**Comment by thevery** [comment]
+:
- Landscape
- Localization
- Caching
- Warning fix

-:
- Photo is pretty small
- Magic string
- Package is not unique

Total:
10 + 3.5 - 2.5 = 11

**Comment by ppzhuk** [comment]
"Photo is pretty small" - I didn't knew it matters.
"Package is not unique" - it was predefined in the repository, so I consciously didn't modify it. Again, didn't knew I should.

**Comment by thevery** [comment]
`Penalty: Неправильное имя пакета` - exactly from task definition

**Pull request closed without merging**